js/client.js

```diff
@@ -1121,6 +1121,8 @@ function toId() {
 				break;
 
 			case 'popup':
+				//storing recieved pop massege for error display on UI when generating link fail
+				window.Storage.prefs("errormassege",data);
```

Comment:
Storing this overall seems ripe for a race condition. I would advise setting a popup listener instead.